gitHooks/scripts/reportViewerPreCommit

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+npx lint-staged
```

Comment:
Same as for java. You should check whether npx is installed

Reply:
Not sure here. I could still change one line without the need for the full npm stack ..